pages/index.js

```diff
@@ -1,21 +1,33 @@
 import Head from "next/head";
 import styled from "styled-components";
 import { Inter } from "next/font/google";
+import { useRouter } from "next/router";
+import useSWR from "swr";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export default function Home() {
+export default function Home({ name, location, image, bewerten, preis }) {
```

Comment:
you should be consistent and even if your page is in german your code should be all in one language - so you should change "bewerten" and "preis" e.g. to "rate" and "price"